python/lsst/fgcmcal/fgcmMakeLut.py

```diff
@@ -534,8 +548,13 @@ def _createLutConfig(self, nCcd):
         # create the common stub of the lutConfig
         lutConfig = {}
         lutConfig['logger'] = self.log
-        lutConfig['filterNames'] = self.config.filterNames
-        lutConfig['stdFilterNames'] = self.config.stdFilterNames
+        physicalFilters = []
+        stdPhysicalFilters = []
+        for physicalFilter in self.config.stdPhysicalFilterMap:
+            physicalFilters.append(physicalFilter)
+            stdPhysicalFilters.append(self.config.stdPhysicalFilterMap[physicalFilter])
```

Comment:
Isn't this the exact same code as above? Seems like a short function would be good.